tables/v1/namespace_associations.sql

```diff
@@ -0,0 +1,25 @@
+SET search_path=v1, public, pg_catalog;
+
+CREATE TABLE IF NOT EXISTS namespace_associations (
+  id                  SERIAL                    NOT NULL  PRIMARY KEY,
+  created_at          TIMESTAMP WITH TIME ZONE  NOT NULL  DEFAULT CURRENT_TIMESTAMP,
+  created_by          TEXT                      NOT NULL,
+  last_modified_at    TIMESTAMP WITH TIME ZONE,
+  last_modified_by    TEXT,
+  parent_namespace_id INTEGER                   NOT NULL  REFERENCES namespaces(id) ON DELETE CASCADE,
+  child_namespace_id  INTEGER                   NOT NULL  REFERENCES namespaces(id) ON DELETE CASCADE,
```

Comment:
Probably want to include update cascading too
```suggestion
  parent_namespace_id INTEGER                   NOT NULL  REFERENCES namespaces(id) ON DELETE CASCADE ON UPDATE CASCADE,
  child_namespace_id  INTEGER                   NOT NULL  REFERENCES namespaces(id) ON DELETE CASCADE ON UPDATE CASCADE,
```